app/src/main/java/com/seoulfitu/android/ui/uimodel/UiAvailabilityFilter.kt

```diff
@@ -0,0 +1,18 @@
+package com.seoulfitu.android.ui.uimodel
+
+enum class UiAvailabilityFilter(val filterName: String) {
+    POSSIBLE("가능"),
+    IMPOSSIBLE("불가능");
+
+    companion object {
+        fun getOptions() = UiAvailabilityFilter.values().map { it.filterName }
```

Comment:
```suggestion
        fun getOptions() = entries.map { it.filterName }
```
IDE가 알려준 방법인데 entries로 써두 되나봐용. 공유 차원 겸 just for fun으로 갖고와봤습니닼ㅋㅋㅋㅋ
아 이건 수정안해도돼요! 왜냐면 나두 `WeatherStatus` 에서 요렇게 안썼기 때문이죠^0^